memberships/templates/memberships/login.html

```diff
@@ -4,17 +4,19 @@
 
       <section class="register_login">
       <div>
-      <h2>Member Login</h2>
+      <h2 class="login_title">Member Login</h2>
 
 
-      <form action="/memberships/login/" method="POST">
+      <form class="login_form"action="/memberships/login/" method="POST">
         {% csrf_token %}
         {{ form.as_p }}
-
+      <div class="loginButton">
```

Comment:
Could we try to keep the class naming scheme consistent. I note that above we are using `snake_case` the most so I guess we should try to stick with that.

Reply:
yes makes sense to keep it consistent